### ISSUE TITLE
[XWAYLAND] Hyprland scares some games into hiding

### ISSUE THREAD

**Issue by TheSunCat** [hyprwm/Hyprland#2685]
When some Xwayland games start, it opens in the bottom right corner of the screen. If I try to focus it (by moving the mouse onto it), it gets focused for one frame then moves out of the way of the mouse, until it is completely offscreen.

https://github.com/hyprwm/Hyprland/assets/44881120/26abeeca-591f-48e5-a2f0-c1bd7e759fcc

The video does not show the mouse because of a bug with OBS or Hyprland (I will report this separately).

**Comment by zakk4223** [comment]
Single display or multiple?

**Comment by vaxerski** [comment]
yeah xwayland weird. Force it to run in wine vd / gamescope and it should be ok

**Comment by TheSunCat** [comment]
> Single display or multiple?

I am using a single display (laptop's built-in 2560x1600 at 1.25 scale).

**Comment by izmyname** [comment]
Bump. Is this still a thing?

**Comment by vaxerski** [comment]
Hello there!

This issue has been closed, as we are moving from issues to discussions. More about this can be found [here](https://wiki.hyprland.org/Contributing-and-Debugging/Issue-Guidelines/) and in #9854.

Firstly, please make sure this issue is still relevant in latest Hyprland. If so, we ask you to open a [discussion](https://github.com/hyprwm/Hyprland/discussions) (please read the discussion guidelines first, in the pinned post)

Thank you for your understanding!
